### ISSUE TITLE
WARNING ABOUT "Zero3K"

### ISSUE THREAD

**Issue by MollySpacefield** [weolar/xpchrome#18]
Weolar, please read this as soon as possible, Zero3K is a messed up person who's harassed people for years on end and you should NOT listen to anything he says/demands and anything of the similar. The ReactOS forum post linked below will have everything you need to know about him. https://reactos.org/forum/viewtopic.php?t=21714

**Comment by weolar** [comment]
This guy looks like a fanatic enthusiast of ReactOS.  :)

**Comment by weolar** [comment]
I actually really like ReactOS as well. I often read the source code of ReactOS and have learned a lot from it

**Comment by MollySpacefield** [comment]
But do you understand what I told you though?

**Comment by Zero3K** [comment]
> Weolar, please read this as soon as possible, Zero3K is a messed up person who's harassed people for years on end and you should NOT listen to anything he says/demands and anything of the similar. The ReactOS forum post linked below will have everything you need to know about him. https://reactos.org/forum/viewtopic.php?t=21714

Then you're not gonna like that I got a a guy to work on the UDFS driver included in it by paying him some money and the AC97 driver included in it is thanks to me asking a developer to fix the AC97 Sample Driver that Microsoft offers and then getting another developer to finish the work he did.

**Comment by MollySpacefield** [comment]
ok https://reactos.org/forum/viewtopic.php?t=21714

**Comment by Daniel7689** [comment]
> Then you're not gonna like that I got a a guy to work on the UDFS driver included in it by paying him some money and the AC97 driver included in it is thanks to me asking a developer to fix the AC97 Sample Driver that Microsoft offers and then getting another developer to finish the work he did.

You have to understand that no one cares about ReactOS. You should cease to harass people over your bug reports. People have actual life outside the computer and aren't gonna be focusing all their attention towards your ReactOS bug reports in a project not even designed for ReactOS.. plus I am pretty sure you scammed the guy you *allegedly* paid.

**Comment by Zero3K** [comment]
I didn't scam the guy I mentioned in regards to the improving of the UDFS driver.

**Comment by Daniel7689** [comment]
> I didn't scam the guy I mentioned in regards to the improving of the UDFS driver.

What is this about then?
![image](https://github.com/weolar/xpchrome/assets/24794949/cce8df55-16d3-4a5c-b478-5f9d7599caf6)

**Comment by Zero3K** [comment]
That was regarding the AC97 driver or something else.

**Comment by jasonliul** [comment]
https://www.reddit.com/r/windowsxp/comments/17x01mo/chromium_115_running_on_windows_xp_without_one/
Zero3K promote xpchrome everywhere, no harm.

**Comment by Zero3K** [comment]
I'm not the one who made that post though.

**Comment by MollySpacefield** [comment]
Yes Jason, but you aren't understanding what Zero3K has done in the past, and STILL continues to do to this day.

**Comment by andika207** [comment]
>Zero3K promote xpchrome everywhere, no harm.

that topic was created by @MWF2

**Comment by Zero3K** [comment]
I've stopped doing it for the most part.
